src/runtime/admin/manageable-component.ts

```diff
@@ -116,7 +120,9 @@ export default class ManageableComponent {
     this.$cwa.admin.componentManager.addToStack({
```

Comment:
I think I saw a flag saying this was un-tested. Just commenting here while I do a manual check too.

Reply:
I think this call to addTostack is untested, unless I'm missing something? Would you mind double checking and adding a test if needed?

Reply:
tests added